layout.css

```diff
@@ -79,44 +79,103 @@
 @media screen and (min-width: 768px) {
     
     #all {
-        width: 960px;
+        width: 768px;
         height: auto;
         margin: auto;
+        border: black 1px solid;
+        

```

Comment:
looks great! stylings seem to be right and everything is in its place. next time you may want to consider making the default styling mobile first and then having just the one media query to change the things that need to be changed to make it desktop styled